en/resources/learning.md

```diff
@@ -56,6 +56,7 @@ self-published, February 2018.
 - [Code with Hugo blog: Express tag](https://codewithhugo.com/tags/express)
 - [Dev.to blog: Express category](https://dev.to/ghvstcode/understanding-express-middleware-a-beginners-guide-g73)
 - [LinuxStans Blog: Express tag](https://linuxstans.com/tag/express-js/)
+- [ButterCMS blog: Express category](https://buttercms.com/blog/tag/node/)
```

Comment:
Thanks for the updated changes! I think you accidentally used the wrong tag here. The link text says "Express category" (which is what we do want to link to from Express site), but then the link itself seems to go to a general "node" tag instead of express.

Reply:
Sorry about this @dougwilson 
I've updated it